alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -407,21 +436,24 @@ private function initMembers()
 			KExternalErrors::dieError(KExternalErrors::MISSING_PARAMETER, self::UI_CONF_ID_PARAM_NAME);
 		
 		// retrieve uiCong Obj
-		$uiConf = uiConfPeer::retrieveByPK($this->uiconfId);
-		if (!$uiConf)
+		$this->uiConf = uiConfPeer::retrieveByPK($this->uiconfId);
```

Comment:
I don't see you added this member to the class similar to what we do for uiConfId:
private $uiconfId = null;